homeassistant/components/mqtt/models.py

```diff
@@ -174,3 +181,31 @@ def async_render_with_possible_json_value(
         return self._value_template.async_render_with_possible_json_value(
             payload, default, variables=values
         )
+
+
+@dataclass
+class MqttData:
```

Comment:
Moving the class from `mixins.py` to `models.py` makes it a lot easier to handle circular dependencies.

Reply:
Yeah, that that should work I think/